packages/devextreme/js/__internal/ui/date_box/m_date_box.mask.parts.ts

```diff
@@ -32,6 +32,7 @@ const PATTERN_GETTERS = {
   m: 'getMinutes',
   s: 'getSeconds',
   S: 'getMilliseconds',
+  x: 'getTimeZoneOffset',
```

Comment:
[mistake, test coverage]

There is a `getTimezoneOffset` Date method, not `getTimeZoneOffset`.
Now if you focus timezone date part in input and press up or down arrow you will get an error in the console. Please, add a test for this case :)

```suggestion
  x: 'getTimezoneOffset',
```

Reply:
[c7b7113](https://github.com/DevExpress/DevExtreme/pull/27770/commits/c7b711321f156783755cf86ccb00120c7846146b)